common/views/components/WorkMedia/WorkMedia.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import {grid, font, spacing} from '../../../utils/classnames';
+import Icon from '../Icon/Icon';
+import Image from '../Image/Image';
+import ImageViewer from '../ImageViewer/ImageViewer';
+import ScrollToInfo from '../ScrollToInfo/ScrollToInfo';
+
+type Props = {|
+  queryString?: string,
+  id: string,
+  trackTitle: string,
+  iiifModel: {
+    contentUrl: string,
+    caption: string,
+    width: string
+  },
+  iiifData: {
+    infoUrl: string,
+    lazyload: boolean,
+    sizesQueries: string
+  }
+|}
+
+const href = (queryString, id) => {
+  return `/works${queryString}#${id}`;
+};
+
+const tracking = (queryString, id, trackTitle) => {
+  return `{
+    "category": "component",
+    "action": "return-to-results:click",
+    "label": "id:${id}, query:${queryString}, title:${trackTitle}"
+  }`;
```

Comment:
You can also `JSON.stringify` JS objects to make sure no errors slip in.

Reply:
We're not (yet) sending this in JSON string format; just key:value pairs.